x/oracle/types/test_common.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+// sdk "github.com/cosmos/cosmos-sdk/types"

```

Comment:
this file doesn't have any test case. Please delete it or add the necessary tests

Reply:
fixed: https://github.com/musnit/peggy/commit/586e80686df398a42e305fc14b58f0ccacb81697